scripts/gogenerate.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
```

Comment:
no comment?  it's hard to maintain later.

Reply:
Added some.